src/stack_aggregator.py

```diff
@@ -586,7 +587,10 @@ def execute(aggregated=None, persist=True):
             manifest = result['details'][0]['manifest_file']
             manifest_file_path = result['details'][0]['manifest_file_path']
             epv_set = create_dependency_data_set(resolved, ecosystem)
-            transitive_count = len(epv_set.get('transitive', []))
+            if show_transitive == "true":
+                transitive_count = len(epv_set.get('transitive', []))
+            else:
+                transitive_count = -1
```

Comment:
@yzainee can you please put some comments in the PR on how the fix solves the problem.

Reply:
ok i will add comments

Reply:
@samuzzal-choudhury comment added.